src/late_multicellular_stage/MulticellularStage.tscn

```diff
@@ -103,4 +103,5 @@ __meta__ = {
 [connection signal="OnOpenInventoryPressed" from="MulticellularHUD" to="PlayerMulticellularInput" method="OpenInventory"]
 [connection signal="OnOpenMenu" from="MulticellularHUD" to="PauseMenu" method="Open"]
 [connection signal="OnOpenMenuToHelp" from="MulticellularHUD" to="PauseMenu" method="OpenToHelp"]
+[connection signal="OnSpeciesFossilised" from="MulticellularHUD/FossilisationDialog" to="." method="UpdateFossilisationButtonStates"]
```

Comment:
Is this connected correctly? The target seems to be the stage and not the HUD.

Reply:
Right, should be fixed now